src/controllers/at-resources.controller.js

```diff
@@ -6,4 +6,35 @@ rscCtrl.dashboard = async(req, res) => {
     res.render("dashboard");
 };
 
+
+
+//-----------Sprint-----------//
+
+// Render Sprint
+rscCtrl.renderSprint = (req, res) => {
+    res.render("sprint");
+};
+
+// Render addSprint
+rscCtrl.renderAddSprintForm = (req, res) =>{
+res.render("/sprint/add");
```

Comment:
Please indent code

Reply:
Do we need to create a form to add a new Sprint?

Reply:
at this point, only on "admin" section

Reply:
Then, do we need to remove or comment this method for this moment ?

Reply:
you can leave it